tests/test_tiff.py

```diff
@@ -49,62 +49,52 @@ def getImage(path, image):
 			print("Importation of images could not be done")
 			print(e)
 			
-def test_img_exists():
+def test_tiff_exists(image_path_tiff):
 	# Test if the file exists, either locally either with the OAC
-	path = "/data/pims/upload_test_tiff/"
-	image = "earthworm-transv-posterior-to-clitellum-02.tiff"
-	getImage(path, image)
-	assert os.path.exists(path+image) == True
-	
-router = APIRouter()
-tile_tags = ['Tiles']
+	get_image(image_path_tiff[0], image_path_tiff[1])
+	assert os.path.exists(os.path.join(image_path_tiff[0],image_path_tiff[1])) == True
 
-def test_info(app, client):
-	# Verify if image format corresponds
-	response = client.get('/image/upload_test_tiff/earthworm-transv-posterior-to-clitellum-02.tiff/info/image')
+def test_tiff_info(client, image_path_tiff):
+	response = client.get(f'/image/upload_test_tiff/{image_path_tiff[1]}/info')
 	assert response.status_code == 200
-	assert "tiff" in response.json()['original_format'].lower()
+	assert "tiff" in response.json()['image']['original_format'].lower()
+	assert response.json()['image']['width'] == 42460
+	assert response.json()['image']['height'] == 29140
 	
-def test_get_tile(app, client):
-	response = client.get("/image/upload_test_tiff/earthworm-transv-posterior-to-clitellum-02.tiff/tile/zoom/3/ti/15", headers={"accept": "image/jpeg"})
+def test_tiff_metadata(client, image_path_tiff):
+	response = client.get(f'/image/upload_test_tiff/{image_path_tiff[1]}/metadata')
 	assert response.status_code == 200
-	img_response = Image.open(io.BytesIO(response.content))
-	width_resp, height_resp = img_response.size
-	assert width_resp == 256
-	assert height_resp == 256
+	assert response.json()['items'][7]["key"] == 'XResolution'
```

Comment:
The assumption that `XResolution` will always be at index 7 is too strong. The order in the items of a collection in this API (and I think we can extend to all Cytomine APIs) is undetermined.